api.go

```diff
@@ -893,10 +882,10 @@ type SearchOptions struct {
 	// sorting matches.
 	UseDocumentRanks bool
 
-	// RanksDampingFactor determines the contribution of documents ranks to the
-	// final ranking based on RRF. A value in (0,1] reduces the contribution,
-	// while a value in (-inf,0) increases it.
-	RanksDampingFactor float64
```

Comment:
On sourcegraph.com we don't roll out zoekt at the same time as frontend. This backwards incomptible change would mean ranking would be a bit broken until it does roll out. However, for engineering simplicity having ranking impacted for a small window of time seems fine to me. Especially since we will fallback to RanksDampingFactor of 0 which does nothing.

Reply:
Oh, I had (incorrectly) that merging this wouldn't have any affect until I bumped a Zoekt version somewhere. Your reasoning makes sense to me too, it seems okay if the ranking for a short period.

Reply:
We manually roll out zoekt (see the bump commits [here](https://github.com/sourcegraph/deploy-sourcegraph-cloud/commits/release/base/indexed-search/indexed-search.StatefulSet.yaml)). So you could time this with a frontend update (there are ways to force that as well).

@stefanhengl should we undo our hacks and rely on normal CI again? That way this won't be as tricky.

Reply:
I see! So here's my plan to minimize breakage (will do this tomorrow):
1. Merge this
2. Update frontend to use the new version of Zoekt
3. Force both to re-deploy at same time